### DISCUSSION TITLE
URL Redirection Problem

### DISCUSSION THREAD

**Discussion by Kas1680** [community/community#65478] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I create a repo with a page and the URL is similar to this: https://abc123.github.io/mysite.io/

The repo have "boxes" that are itself link to another .HTML under the HTML folder. 

It works fine when I was working on my local repo with vscode but when I push it, the URL would change to:

https://abc123.github.io/html/page2.html

This gives me a 404 error. I tried using an if condition to set a base URL and it won't work. Anyone know a way around this?

**Comment by seyLu** [comment]
#### This is already solved right? I checked your commit history, and you've already solved it yourself via absolute paths.

href="html/page2.html" instead of href="../html/page2.html"

#### But if you want more control on routing + add more ui interactivity, then you can maybe check out static site generators like astro, gatsby, hugo, jekyll, to name a few. And then add an automated workflow that builds & deploys the site on a `gh-pages` branch to `<repo>.github.io`.
